function_runtime.py

```diff
@@ -2,6 +2,11 @@
 
 
 def timer(f, *args):
+    """
+    :param f:function to calculate the time on execute
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names